metadata-db-app/src/migrations/052_add_coll_ids_catalog_acls.clj

```diff
@@ -0,0 +1,70 @@
+(ns migrations.052-add-coll-ids-catalog-acls
+  (:require [clojure.java.jdbc :as j]
+            [clojure.edn :as edn]
+            [cmr.common.util :as util]
+            [config.migrate-config :as config]
+            [config.mdb-migrate-helper :as h]
+            [cmr.metadata-db.data.oracle.providers :as providers]
+            [clojure.string :as string]))
+
+(defn- get-concept-ids
+  [provider-id collection-identifier]
+  (let [entry-titles (get collection-identifier :entry-titles)
+        entry-titles (if (seq entry-titles)
```

Comment:
You could change this line to `(when (seq entry-titles))` or on line  13 do a `(when-let (seq entry-titles))` and could get rid of line 16.